src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java

```diff
@@ -9,32 +9,53 @@
 import edu.kis.legacy.drawer.panel.DrawPanelController;
 import edu.kis.legacy.drawer.shape.LineFactory;
 import edu.kis.powp.appbase.Application;
-import edu.kis.powp.jobs2d.drivers.adapter.BoxDrawerAdapter;
+import edu.kis.powp.jobs2d.drivers.adapter.DrawerDriverAdapter;
 import edu.kis.powp.jobs2d.drivers.adapter.LineDrawerAdapter;
 import edu.kis.powp.jobs2d.events.SelectChangeVisibleOptionListener;
 import edu.kis.powp.jobs2d.events.SelectTestFigureOptionListener;
 import edu.kis.powp.jobs2d.features.DrawerFeature;
 import edu.kis.powp.jobs2d.features.DriverFeature;
 
+
+
+
 public class TestJobs2dPatterns {
 	private final static Logger logger = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
 
+	public enum TestFigureOption {
+		OPTION_1(1),
+		OPTION_2(2),
+		OPTION_3(3),
+		OPTION_4(4);
```

Comment:
SelectTestFigureOptionListener does not use these constants. Changes here are not consistent with the listener class.
Enums should have better/meaningful names.